resources/views/frontend/catalog-new/index-new.blade.php

```diff
@@ -696,7 +697,15 @@ class="tw-h-4 tw-w-4 tw-fill-current" viewBox="0 0 256 256">
                     </div>
                     <div class="tw-min-h-screen">
                         <div class="tw-px-4 md:tw-px-16 md:tw-py-10">
-                            <div v-if="artworks.length === 0"
+                            <div v-if="hasError"
+                                class="tw-flex tw-w-full tw-flex-col tw-items-center tw-justify-center tw-py-40 tw-text-lg">
+                                <span>Uuups, niečo sa pokazilo :( skús</span>
+                                <reload-controller v-slot="rc">
+                                    <button class="tw-inline-block tw-underline" @click="rc.reload">obnoviť
+                                        stránku.</button>
+                                </reload-controller>
```

Comment:
This is fine and consistent, but it could've just been a simple `<reload-button />` component. There is nothing to "control" here
```suggestion
                                <reload-button class="tw-inline-block tw-underline">
                                  obnoviť stránku.
                                </reload-button>
```